earthpy/tests/test_extent_to_json.py

```diff
@@ -0,0 +1,56 @@
+""" Tests for the extent_to_json function """
+
+
+import pandas as pd
+import pytest
+import geopandas as gpd
+from shapely.geometry import Polygon, Point
+import earthpy.spatial as es
+
+
+@pytest.fixture
+def list_out():
+
+    return es.extent_to_json([0, 0, 1, 1])
+
+
+def test_list_format_works():
```

Comment:
Can this test could use the `list_out` fixture you created?
```suggestion
def test_list_format_works(list_out):
```

Reply:
what do you think about the var name `list_out` @mbjoseph it just occurred to me that it's pretty darn uninformative / generic.